src/Examples/Flowsheets/hda_flowsheets.py

```diff
@@ -0,0 +1,471 @@
+    print('Setting inputs...')
+    print()
+
+    m.fs.M101.toluene_feed.flow_mol_phase_comp[0, "Vap", "benzene"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.toluene_feed.flow_mol_phase_comp[0, "Vap", "toluene"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.toluene_feed.flow_mol_phase_comp[0, "Vap", "hydrogen"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.toluene_feed.flow_mol_phase_comp[0, "Vap", "methane"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.toluene_feed.flow_mol_phase_comp[0, "Liq", "benzene"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.toluene_feed.flow_mol_phase_comp[0, "Liq", "toluene"].fix(0.30*pyunits.mol/pyunits.s)
+    m.fs.M101.toluene_feed.flow_mol_phase_comp[0, "Liq", "hydrogen"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.toluene_feed.flow_mol_phase_comp[0, "Liq", "methane"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.toluene_feed.temperature.fix(303.2*pyunits.K)
+    m.fs.M101.toluene_feed.pressure.fix(350000*pyunits.Pa)
+
+    m.fs.M101.hydrogen_feed.flow_mol_phase_comp[0, "Vap", "benzene"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.hydrogen_feed.flow_mol_phase_comp[0, "Vap", "toluene"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.hydrogen_feed.flow_mol_phase_comp[0, "Vap", "hydrogen"].fix(0.30*pyunits.mol/pyunits.s)
+    m.fs.M101.hydrogen_feed.flow_mol_phase_comp[0, "Vap", "methane"].fix(0.02*pyunits.mol/pyunits.s)
+    m.fs.M101.hydrogen_feed.flow_mol_phase_comp[0, "Liq", "benzene"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.hydrogen_feed.flow_mol_phase_comp[0, "Liq", "toluene"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.hydrogen_feed.flow_mol_phase_comp[0, "Liq", "hydrogen"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.hydrogen_feed.flow_mol_phase_comp[0, "Liq", "methane"].fix(1e-5*pyunits.mol/pyunits.s)
+    m.fs.M101.hydrogen_feed.temperature.fix(303.2*pyunits.K)
+    m.fs.M101.hydrogen_feed.pressure.fix(350000*pyunits.Pa)
+
+    m.fs.H101.outlet.temperature.fix(600*pyunits.K)
+
+    m.fs.R101.conversion = Var(initialize=0.75, bounds=(0, 1))
+
+    m.fs.R101.conv_constraint = Constraint(
+        expr=m.fs.R101.conversion*m.fs.R101.inlet.
+        flow_mol_phase_comp[0, "Vap", "toluene"] ==
+        (m.fs.R101.inlet.flow_mol_phase_comp[0, "Vap", "toluene"] -
+         m.fs.R101.outlet.flow_mol_phase_comp[0, "Vap", "toluene"]))
+
+    m.fs.R101.conversion.fix(0.75*pyunits.dimensionless)
+    m.fs.R101.heat_duty.fix(0*pyunits.W)
+
+    m.fs.F101.vap_outlet.temperature.fix(325.0*pyunits.K)
+    m.fs.F101.deltaP.fix(0*pyunits.Pa)
+
+    m.fs.F102.vap_outlet.temperature.fix(375*pyunits.K)
+    m.fs.F102.deltaP.fix(-200000*pyunits.Pa)
+
+    m.fs.S101.split_fraction[0, "purge"].fix(0.2)
+    m.fs.C101.outlet.pressure.fix(350000*pyunits.Pa)
+
+    # initialize flowsheet
+    print('Initializing flowsheet...')
+    print()
+
+    seq = SequentialDecomposition()
+    seq.options.select_tear_method = "heuristic"
+    seq.options.tear_method = "Wegstein"
+    seq.options.iterLim = 5
```

Comment:
Can we reduce this to something smaller (maybe 3)? Keeping the initialization time to a minimum is always good.